src/index.js

```diff
@@ -1,33 +1,89 @@
+/* 
+  ED - edited code
+  NEW - new code
+
+  Improvements:
```

Comment:
you can put some comment like a description for your work, but never use comments in code with explanation what it does!